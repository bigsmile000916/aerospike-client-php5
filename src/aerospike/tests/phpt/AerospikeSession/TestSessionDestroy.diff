001+ Fatal error: Uncaught exception 'ReflectionException' with message 'Method testSessionDestroy does not exist' in /home/gslab/aerospike-client-php/src/aerospike/tests/astestframework/ASTestFramework.inc:192
001- ERR_RECORD_NOT_FOUND
002+ Stack trace:
003+ #0 /home/gslab/aerospike-client-php/src/aerospike/tests/astestframework/ASTestFramework.inc(192): ReflectionClass->getMethod('testSessionDest...')
004+ #1 /home/gslab/aerospike-client-php/src/aerospike/tests/astestframework/astest-phpt-loader.inc(28): ASTestFramework->runSingleTest('testSessionDest...')
005+ #2 /home/gslab/aerospike-client-php/src/aerospike/tests/phpt/AerospikeSession/TestSessionDestroy.php(3): aerospike_phpt_runtest('AerospikeSessio...', 'testSessionDest...')
006+ #3 {main}
007+   thrown in /home/gslab/aerospike-client-php/src/aerospike/tests/astestframework/ASTestFramework.inc on line 192