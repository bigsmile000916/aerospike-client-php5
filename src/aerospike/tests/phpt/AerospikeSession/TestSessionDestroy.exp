ERR_RECORD_NOT_FOUND