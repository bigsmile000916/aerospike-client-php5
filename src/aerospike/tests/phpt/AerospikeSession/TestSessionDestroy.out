Fatal error: Uncaught exception 'ReflectionException' with message 'Method testSessionDestroy does not exist' in /home/gslab/aerospike-client-php/src/aerospike/tests/astestframework/ASTestFramework.inc:192
Stack trace:
#0 /home/gslab/aerospike-client-php/src/aerospike/tests/astestframework/ASTestFramework.inc(192): ReflectionClass->getMethod('testSessionDest...')
#1 /home/gslab/aerospike-client-php/src/aerospike/tests/astestframework/astest-phpt-loader.inc(28): ASTestFramework->runSingleTest('testSessionDest...')
#2 /home/gslab/aerospike-client-php/src/aerospike/tests/phpt/AerospikeSession/TestSessionDestroy.php(3): aerospike_phpt_runtest('AerospikeSessio...', 'testSessionDest...')
#3 {main}
  thrown in /home/gslab/aerospike-client-php/src/aerospike/tests/astestframework/ASTestFramework.inc on line 192